CaseStudy.md

```diff
@@ -0,0 +1,166 @@
+# Case Study – “Accelerating ML Model Development and Deployment on the edge with the Machine Learning Model Factory”
+
+
+## Table of Contents
+* [Purpose](#purpose)
+* [Customer Overview](#customer_overview)
+* [Problem Statement](#problem_statement)
+* [Solution Overview](#solution_overview)
+     * [Machine Learning Factory Model](machine_learning_factory_model)
+     * [Use Case Builder](use_case_builder)
+     * [Use Case Deployment](use_case_deployment)
+* [Technical Implementation](technical_implementation)
+     * [Model Factory Design](model_factory_design)
+     * [Use Case Builder Design](use_case_builder_design)
+     * [Data Collection on the Edge Design](data_collection_on_the_edge_design)
+     * [Deployment Size Optimizations for Constrained Network](deployment_size_optimizations_for_constrained_network)
+        * [Orchestrated Inferencing](Orchestrated_Inferencing)
+        * [Docker image optimizations](Docker_image_optimizations)
+     * [Deployment Size Reduction Data](Deployment_Size_Reduction_Data)
+ * [Conclusion](conclusion)
+ * [References](references)
+
+<!-- You can include any other section that is pertinent to your problem -->
+### Document Control:
```

Comment:
We can remove this section

Reply:
Removed ToC and Document Control sections as advised